xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2621,6 +2621,7 @@ xe.invitation.displayMessageTable.recipient=Email
 xe.invitation.displayMessageTable.history=Message History
 xe.invitation.displayMessageTable.showHistory=Show History
 xe.invitation.displayMessageTable.multipleRecipients={0} Recipients
+xe.invitation.displayMessageTable.checkbox.input.label=Checkbox message n.[{0}]
```

Comment:
This label seems strange, in particular when it is displayed. Do you have a screenshot how this looks?

Reply:
I forgot to put the class 'sr-only' on the label. I tested out and found out how this kind of checkbox was used.

This is  the checkbox at the start of every line on this table:
![20997-Result](https://github.com/xwiki/xwiki-platform/assets/28761965/d37e13c9-de91-4e9f-88d1-121776613ca7)


Updated the translation value from 'Checkbox message n.[{0}]' to 'Select this invitation'.
Addressed in https://github.com/xwiki/xwiki-platform/pull/2249/commits/dec92b6d0892157a3daa8de40289e7aa56b08208.